FirebaseAuth/Sources/Swift/Utilities/AuthRecaptchaVerifier.swift

```diff
@@ -15,6 +15,10 @@
 #if os(iOS)
 
   import Foundation
+
+  #if !COCOAPODS
```

Comment:
Maybe this should be `SWIFT_PACKAGE` so as to not be true for the zip build, where `FirebaseAuthInternal` won't exist.